Enterprise-CMCS/macpro-onemac
# Oy2 26063 - RAI Response Withdraw Enabled is no longer a package status

Story: https://qmacbis.atlassian.net/browse/OY2-26063
Endpoint: See github-actions bot comment

### Details

RAI Response Withdraw Enabled is no longer a package status. When the enable withdraw action is taken the package status should remain in Pending/Under Review. Instead there will be a "substatus" displayed under the status on both the dashboard and the details page.

### Changes

- removed newStatus from enable and disable withdraw config since they are not actually used
- removed setting the eventTimestamp and refactored buildAnyPackage to not rely on that
- refactored buildAnyPackage to detect substatus and lock in package status

1. what fields are new
The package item will now have a new field named substatus. This will drive action logic and display of substatus on pacakge details and package list screens
2. what fields are removed
None
3. what fields are going to hold different content
package status will no longer be able to hold "RAI Response Withdraw Enabled" status and will instead stay in Pending

### Test Plan

1. Login as CMS user
2. Find any package in Pending status with a submitted RAI
5. Click the Enable Rai Withdraw action
6. Verify package status remains Pending and substatus shows underneath
7. Navigate back to package list and verify package has both status and substatus in table
8. Navigate back to detail view for package and select disable withdraw rai response
9. Verify package still remains in original status (Pending)
10. enable withdraw rai again
11. login as state user and request withdraw
12. verify status changes to xxxxxxxxx
